cluster/examples/cache/rediscluster/gcp/resource-class.yaml

```diff
@@ -2,24 +2,25 @@
 apiVersion: cache.crossplane.io/v1alpha1
 kind: RedisClusterClass
 metadata:
-  name: redis-standard
-  namespace: app-project1-dev
+  name: redis-standard-2
+  namespace: myappns
```

Comment:
Are we wanting to move off the app namespace pattern for all examples? (I know it is probably less important with cluster scoped managed resources)

Reply:
Oh, no. These two lines are remnant of my experiments, just corrected. Thanks for catching!